HEC.FDA.Model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -65,8 +63,8 @@ public class ImpactAreaStageDamage : PropertyValidationHelper, IDontImplementVal
         public Inventory Inventory { get; }
         public int ImpactAreaID { get; }
 
-        public event ProgressReportedEventHandler ProgressReport;
-        public event MessageReportedEventHandler MessageReport;
+        public event MVVMFramework.Base.Events.ProgressReportedEventHandler ProgressReport;
```

Comment:
I don't believe we actually use these events. We've been trying to rip them out piecewise for awhile. Confirm whether these are used, and if not, delete them.